### DISCUSSION TITLE
Duda sobre términos

### DISCUSSION THREAD

**Discussion by erikgiovani** [reactjs/es.reactjs.org#603] [Q&A] [answered]
Hola, tengo la duda de si traducir los términos **Lifting state up** y **prop drilling**

**Comment by carburo** [comment]
Estas serían mis recomendaciones, aunque me gustaría ver otras opiniones:
- «Lifting state up» se ha solido traducir como «levantar el estado».
- «prop drilling» podría dejarse sin traducir en cursiva *«prop drilling»* con una traducción literal entre paréntesis en su primer uso: *«prop drilling»* (‘perforación de props’)

**Comment by carburo** [comment, marked as answer]
Las traducciones ya se añadieron en la wiki
